src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -1469,7 +1469,7 @@ FullWidthRepeat:
         End Function
 
         Private Function TokenOfStringCached(spelling As String) As SyntaxKind
-            If spelling.Length = 1 OrElse spelling.Length > 16 Then
+            If spelling.Length > 16 Then
```

Comment:
What besides the \#r is going to be affected by this change? Do we really want this to happen?